SkietbaanBE/SkietbaanBE/Helper/NotificationMessages.cs

```diff
@@ -75,6 +77,48 @@ public void GroupNotification(ModelsContext _context, Group group, User user)
                 message = e.Message;
             }
         }
+
+        public void LOS(string token)
+        {
+            var user = _context.Users.SingleOrDefault(x => x.Token == token);
+            var notification = new Notifications
+            {
+                User = user,
+                IsRead = false,
+                TypeOfNotification = "Document",
+                NotificationMessage = "Hello " + user.Username + " Letter of Status received. Check your documents"
+            };
+            _context.Add(notification);
```

Comment:
Due to the lack of error handling while having a db interaction will break your application if there are any connection errors